package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "lovefield",
   "version": "2.0.62",
+  "main": "dist/lovefield.js",
```

Comment:
Usually main points to the development version. Example. https://github.com/jashkenas/underscore/blob/master/package.json